anyblok/config.py

```diff
@@ -778,15 +795,24 @@ def add_plugins(group):
     """
     group.add_argument('--registry-cls', dest='Registry', type=AnyBlokPlugin,
                        default='anyblok.registry:Registry',
-                       help="Registry class to use")
+                       help="Registry class to use",
+                       deprecated=("This plugins will be removed in version "
+                                   "2.0, because this behaviours is "
+                                   "dangerous"))
```

Comment:
[anyblok fast API](https://github.com/AnyBlok/AnyBlok_FastAPI/blob/master/anyblok_fastapi/config.py#L253) were using it to store http routes declared by bloks !

I gonna find an other way to do it ! ;-)

Reply:
From anyblok.registry import Registry ;)